% Template for PLoS
% Version 3.7 Aug 2025
%
% % % % % % % % % % % % % % % % % % % % % %
%
% -- IMPORTANT NOTE
%
% This template contains comments intended 
% to minimize problems and delays during our production 
% process. Please follow the template instructions
% whenever possible.
%
% % % % % % % % % % % % % % % % % % % % % % % 
%
% Once your paper is accepted for publication, 
% PLEASE REMOVE ALL TRACKED CHANGES in this file 
% and leave only the final text of your manuscript. 
% PLOS recommends the use of latexdiff to track changes during review, as this will help to maintain a clean tex file.
% Visit https://www.ctan.org/pkg/latexdiff?lang=en for info or contact us at latex@plos.org.
%
%
% There are no restrictions on package use within the LaTeX files except that no packages listed in the template may be deleted.
%
% Please do not include colors or graphics in the text.
%
% The manuscript LaTeX source should be contained within a single file (do not use \input, \externaldocument, or similar commands).
%
% % % % % % % % % % % % % % % % % % % % % % %
%
% -- FIGURES AND TABLES
%
% Please include tables/figure captions directly after the paragraph where they are first cited in the text.
%
% DO NOT INCLUDE GRAPHICS IN YOUR MANUSCRIPT
% - Figures should be uploaded separately from your manuscript file. 
% - Figures generated using LaTeX should be extracted and removed from the PDF before submission. 
% - Figures containing multiple panels/subfigures must be combined into one image file before submission.
% For figure citations, please use "Fig" instead of "Figure".
% See http://journals.plos.org/plosone/s/figures for PLOS figure guidelines.
%
% Tables should be cell-based and may not contain:
% - spacing/line breaks within cells to alter layout or alignment
% - do not nest tabular environments (no tabular environments within tabular environments)
% - no graphics or colored text (cell background color/shading OK)
% See http://journals.plos.org/plosone/s/tables for table guidelines.
%
% For tables that exceed the width of the text column, use the adjustwidth environment as illustrated in the example table in text below.
%
% % % % % % % % % % % % % % % % % % % % % % % %
%
% -- EQUATIONS, MATH SYMBOLS, SUBSCRIPTS, AND SUPERSCRIPTS
%
% IMPORTANT
% Below are a few tips to help format your equations and other special characters according to our specifications. For more tips to help reduce the possibility of formatting errors during conversion, please see our LaTeX guidelines at http://journals.plos.org/plosone/s/latex
%
% For inline equations, please be sure to include all portions of an equation in the math environment.  For example, x$^2$ is incorrect; this should be formatted as $x^2$ (or $\mathrm{x}^2$ if the romanized font is desired).
%
% Do not include text that is not math in the math environment. For example, CO2 should be written as CO\textsubscript{2} instead of CO$_2$.
%
% Please add line breaks to long display equations when possible in order to fit size of the column. 
%
% For inline equations, please do not include punctuation (commas, etc) within the math environment unless this is part of the equation.
%
% When adding superscript or subscripts outside of brackets/braces, please group using {}.  For example, change "[U(D,E,\gamma)]^2" to "{[U(D,E,\gamma)]}^2". 
%
% Do not use \cal for caligraphic font.  Instead, use \mathcal{}
%
% % % % % % % % % % % % % % % % % % % % % % % % 
%
% Please contact latex@plos.org with any questions.
%
% % % % % % % % % % % % % % % % % % % % % % % %

\documentclass[10pt,letterpaper]{article}
\usepackage[top=0.85in,left=2.75in,footskip=0.75in]{geometry}

% amsmath and amssymb packages, useful for mathematical formulas and symbols
\usepackage{amsmath,amssymb}

% Use adjustwidth environment to exceed column width (see example table in text)
\usepackage{changepage}

% textcomp package and marvosym package for additional characters
\usepackage{textcomp,marvosym}

% cite package, to clean up citations in the main text. Do not remove.
\usepackage{cite}

% Use nameref to cite supporting information files (see Supporting Information section for more info)
\usepackage{nameref,hyperref}

% line numbers
\usepackage[right]{lineno}

% ligatures disabled
\usepackage[nopatch=eqnum]{microtype}
\DisableLigatures[f]{encoding = *, family = * }

% color can be used to apply background shading to table cells only
\usepackage[table]{xcolor}

% array package and thick rules for tables
\usepackage{array}

% create "+" rule type for thick vertical lines
\newcolumntype{+}{!{\vrule width 2pt}}

% create \thickcline for thick horizontal lines of variable length
\newlength\savedwidth
\newcommand\thickcline[1]{%
  \noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
  \cline{#1}%
  \noalign{\vskip\arrayrulewidth}%
  \noalign{\global\arrayrulewidth\savedwidth}%
}

% \thickhline command for thick horizontal lines that span the table
\newcommand\thickhline{\noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
\hline
\noalign{\global\arrayrulewidth\savedwidth}}


% Remove comment for double spacing
%\usepackage{setspace} 
%\doublespacing

% Text layout
\raggedright
\setlength{\parindent}{0.5cm}
\textwidth 5.25in
\textheight 8.75in

% Bold the 'Figure #' in the caption and separate it from the title/caption with a period
% Captions will be left justified
\usepackage[aboveskip=1pt,labelfont=bf,labelsep=period,justification=raggedright,singlelinecheck=off]{caption}
\renewcommand{\figurename}{Fig}

% Please use the included `plos2025.bst` as your BibTeX style
\bibliographystyle{plos2025}

% Remove brackets from numbering in List of References
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother



% Header and Footer with logo
\usepackage{lastpage,fancyhdr,graphicx}
\usepackage{epstopdf}
%\pagestyle{myheadings}
\pagestyle{fancy}
\fancyhf{}
%\setlength{\headheight}{27.023pt}
%\lhead{\includegraphics[width=2.0in]{PLOS-submission.eps}}
\rfoot{\thepage/\pageref{LastPage}}
\renewcommand{\headrulewidth}{0pt}
\renewcommand{\footrule}{\hrule height 2pt \vspace{2mm}}
\fancyheadoffset[L]{2.25in}
\fancyfootoffset[L]{2.25in}
\lfoot{\today}

%% Include all user-defined macros below

\newcommand{\lorem}{\textbf{LOREM}}
\newcommand{\ipsum}{\textbf{IPSUM}}

%% END MACROS SECTION


\begin{document}
\vspace*{0.2in}

% Title must be 250 characters or less.
\begin{flushleft}
{\Large
\textbf{A century of weekly notifiable disease incidence data by province in Canada} % Please use "sentence case" for title and headings (capitalize only the first word in a title (or heading), the first word in a subtitle (or subheading), and any proper nouns).
}
\newline
% Insert author names, affiliations and corresponding author email (do not include titles, positions, or degrees).
\\
David J. D. Earn\textsuperscript{1,2},
Gabrielle MacKinnon\textsuperscript{3},
Samara Manzin\textsuperscript{4},
Michael Roswell\textsuperscript{5},
Steve Cygu\textsuperscript{6},
Chyunfung Shi\textsuperscript{7},
Benjamin M. Bolker\textsuperscript{1,7},
Jonathan Dushoff\textsuperscript{2,7},
Steven C. Walker\textsuperscript{1*}
\\
\bigskip
\textbf{1} Department of Mathematics and Statistics, McMaster University, Hamilton, Ontario, Canada
\\
\textbf{2} M.\,G.\ DeGroote Institute for Infectious Disease Research, McMaster University, Hamilton, Ontario, Canada
\\
\textbf{3} Department of Mathematics and Statistics, McGill University, Montr\'{e}al, Qu\'{e}bec, Canada
\\
\textbf{4} Department of Biology, McGill University, Montr\'{e}al, Qu\'{e}bec, Canada
\\
\textbf{5} Department of Biology, University of Maryland, College Park, Maryland, United States of America
\\
\textbf{6} African Population Health and Research Center, Nairobi, Kenya
\\
\textbf{7} Department of Biology, McMaster University, Hamilton, Ontario, Canada
\\
\bigskip

% Insert additional author notes using the symbols described below. Insert symbol callouts after author names as necessary.
% 
% Remove or comment out the author notes below if they aren't used.
%
% Primary Equal Contribution Note
%\Yinyang These authors contributed equally to this work.

% Additional Equal Contribution Note
% Also use this double-dagger symbol for special authorship notes, such as senior authorship.
%\ddag These authors also contributed equally to this work.

% Current address notes
%\textcurrency Current Address: Dept/Program/Center, Institution Name, City, State, Country % change symbol to "\textcurrency a" if more than one current address note
% \textcurrency b Insert second current address 
% \textcurrency c Insert third current address

% Deceased author note
%\dag Deceased

% Group/Consortium Author Note
%\textpilcrow Membership list can be found in the Acknowledgments section.

% Use the asterisk to denote corresponding authorship and provide email address in note below.
* swalk@mcmaster.ca

\end{flushleft}
% Please keep the abstract below 300 words

% For PLOS Medicine research article authors, please structure your abstract
% with "Background", "Method and Findings" and "Conclusion" sections per
% journal requirements.

% For PLOS Neglected Tropical Diseases research article authors, please
% structure your abstract with "Background", "Methodology", "Findings", and
% "Conclusion" sections per journal requirements.
%
\section*{Abstract}
Canadian notifiable disease surveillance programmes have recorded communicable disease incidence data, dating back to the late 19th century. A Public Health Agency of Canada web-portal provides summaries of these data from 1924--2023, but lacks details on how incidence varies seasonally and geographically among provinces. The sub-annual (weekly, monthly, quarterly) and sub-national (provincial, territorial) data required to study such patterns appear in government documents, but are only available in typewritten or handwritten hard copies.

We digitized and collated these sources to make sub-annual and sub-national Canadian disease incidence data conveniently available for researchers. We manually transcribed hard copies into digital spreadsheets resembling the originals, enabling accurate transcription through easier cross-checking. We supplemented these historical data sources with more recent digital spreadsheets obtained directly from two provincial agencies. We standardized and combined these spreadsheets into consistent, machine-readable CSV files containing \input{unharmonized_sample_size}\ incidence values from 1903--2021.

Because multiple publications and agencies reproduced case counts from the same surveillance system, and many publications reported the same cases at multiple levels of aggregation, many cases were counted more than once among these \input{unharmonized_sample_size}\ incidence values. We reconciled overlapping counts to produce a dataset containing \input{normalized_sample_size}\ unique incidence values at sub-national and sub-annual scales (\input{normalized_sample_size_wk}\ weekly; \input{normalized_sample_size_2wk}\ 2-weekly; \input{normalized_sample_size_mo}\ monthly; \input{normalized_sample_size_qr}\ quarterly; \input{normalized_sample_size_3qr}\ 3-quarterly) covering \input{n_basal_diseases}\ diseases stratified by province or territory. We illustrate the value of these sub-annual and sub-national data using two examples: synchronized annual cycles of poliomyelitis across provinces and spatially heterogeneous resurgence of whooping cough in the 1990s.

Canada's infectious disease surveillance has produced a detailed record of sub-annual and sub-national disease incidence data that remains largely unexplored. This record is now available as the Canadian Notifiable Disease Incidence Dataset (CANDID), hosted on a publicly accessible website along with code to reproduce it, and scans of the original sources.

%\linenumbers

\section*{Introduction}

Learning from data on past communicable disease outbreaks and recurrent epidemics is an important component of public health planning~\cite{AndeMay91,Earn+00,BaucEarn03,HempEarn15,ogden2024mathematical}. The year 2024 marked the 100th anniversary since the Canadian federal government began collecting such data through notifiable disease surveillance programmes~\cite{summaryNotifiable1954,sockett1996communicable,carter1991establishing,doherty2000establishing,totten2019updates}. Several provinces conducted surveillance before 1924 going back to the late 19th century, although here we focus on the period from 1903 to 2021. The Public Health Agency of Canada (PHAC) provides summaries of these data as annual, national totals since 1924 through an online portal (\url{https://diseases.canada.ca/notifiable}). These coarsely aggregated data are useful for understanding broad trends, but provide no information on seasonal patterns of incidence within years or on spatial patterns across provinces. For example, annual data cannot be used to estimate the timing and shape of an outbreak curve, and national data cannot be used to assess whether provinces displayed different patterns of spread in the vaccine era.

The Canadian notification system has at times collected more informative weekly, monthly, and quarterly incidence data, stratified by disease and province/territory, but these data are not available through the portal. We use the term sub-annual to refer to surveillance data reported at a finer temporal resolution than one year (e.g., weekly, monthly, or quarterly), as opposed to annual data aggregated over the full year. Likewise, we use the term sub-national to refer to data reported at a finer spatial resolution than the national level (e.g., by province or territory). These higher-resolution data enable investigation of variation within years and across regions, and can be aggregated to coarser temporal or spatial resolutions when broader comparisons are needed. The reverse, however, is not possible, making higher-resolution data inherently more versatile.

It has generally been prohibitively challenging for researchers to find and access these sub-annual and sub-national data. Still, we show here that much of this information exists in government publications (either as hard copy reference material or online), in data and documents obtained directly from government agencies, and in provincial publications, some of which predate the establishment of the federal system in 1924. This information appeared under a variety of evolving titles often issued by Statistics Canada (and its predecessor the Dominion Bureau of Statistics), Health Canada, and provincial and territorial health departments. The diversity of publication formats and naming conventions has made systematic retrieval challenging. To address this, we have documented the available resources in this literature (see S1.1 in \nameref{S1_Appendix}) and digitized the data they contain using open-source tools that we developed. Here, we introduce CANDID (Canadian Disease Incidence Dataset), a curated dataset that integrates and cleans these disparate data sources to create a comprehensive and accessible digital record of sub-annual (weekly, monthly, quarterly) and sub-national (provincial, territorial) Canadian notifiable disease incidence data--including substantial amounts of data that were publicly available in principle but almost entirely overlooked. Our objective is to make these sub-annual and sub-national disease incidence data, which have been difficult to obtain, conveniently available for public health and research use.

With this paper we announce the existence of CANDID on a publicly available web site (S1.3 in \nameref{S1_Appendix}) and illustrate its value. Two examples illustrate the advantages of the sub-annual and sub-national data provided by CANDID. First, we describe how poliomyelitis incidence was strongly and consistently seasonal from \input{polio-min-year-peak} to \input{polio-max-year-peak}, and that the yearly peaks were synchronous across provinces. This pronounced synchrony contrasts with patterns in the United States~\cite{martinez2015unraveling}, but can be interpreted within the context of a broader latitudinal gradient in the timing of seasonal epidemic peaks across North America. Second, the well-studied apparent resurgence of whooping cough in the 1990s showed significant regional variation. While the territories, prairie provinces, and Qu\'ebec experienced clear increases in incidence, this pattern was not evident in British Columbia, Ontario, and the Atlantic region. We use a simple graphical approach in these examples, illustrating how these data can inform fundamental questions in epidemiology. Formal statistical analyses that dig deeper into specific questions will follow in subsequent publications. The Canadian Disease Incidence Dataset (CANDID) has the potential to drive many studies by the broader public health research community.

\section*{Materials and methods}
\subsection*{Data sources}\label{sec:data-sources}

We began searching for Canadian historical infectious disease notification data in 2000. We focused on sub-annual and sub-national data collected through the surveillance programs described in Section S1.1 in \nameref{S1_Appendix}, particularly incidence over entire provinces and territories, since finer spatial resolution data were rare. We compiled provincial and territorial population data \cite{canada1925sixth,statistics1973population,statcan2021population} to compute comparable incidence rates per 100,000. We acquired data in any of three formats: (1) paper hard copies, (2) digitally produced PDF files, or (3) spreadsheets (including CSV files). We scanned all of the hard copies. We found that Optical Character Recognition (OCR) was unable to convert scans into digital spreadsheets with sufficient accuracy, so we entered the data manually (see the \nameref{sec:data-entry} section). We used PDF extraction tools \cite{PDFTables} to avoid manual entry of digitally-produced PDF pages.

\subsection*{Data entry}\label{sec:data-entry}

We manually entered the information in scans into replica Excel spreadsheets (i.e., digital spreadsheets in which the layout of each spreadsheet matches the original), to facilitate comparing reproductions with their sources (Fig~\ref{fig:typewritten-eg}). The Ontario Ministry of Health data were entered before our comprehensive effort and were only partially entered as replicas (they are the sole exception). Reading scans and interpreting handwritten sources (Fig~\ref{fig:handwritten-eg}) was often a slow, and occasionally error-prone, process. Where available in these sources, we also entered reported annual and national data to support later validation against marginal totals (see the \nameref{sec:quality-control} section). When we encountered unclear numbers in the source material, we recorded our initial guesses using a predetermined format described in Section S1.4.1 in \nameref{S1_Appendix}, so that they could be processed systematically. These guesses were revisited and refined as needed when validation procedures, which compared sums of incidence values with marginal totals reported in both CANDID sources and the PHAC portal \cite{totten2019updates}, revealed discrepancies. Scripted data preparation pipelines (see the \nameref{sec:data-prep} section) facilitated updates throughout this process. We provide details on our data entry process in Section S1.4.1 in \nameref{S1_Appendix}.

\begin{figure}[!h]
  \caption{\textbf{Example of a typewritten source document prepared using a typewriter.} The top panel shows a scan of part of this source document, and the bottom panel shows our replica in Microsoft Excel of this same part.}
  \label{fig:typewritten-eg}
\end{figure}

\begin{figure}[!h]
  \caption{\textbf{Examples of handwritten data}. Most handwritten hard copies, such as the 1939 erysipelas and gonococcal data from the Ontario Ministry of Health (A), are easy to read. Others, like the 1955 poliomyelitis data from Statistics Canada (B), are difficult to read, posing challenges for digitization.}
  \label{fig:handwritten-eg}
\end{figure}

\subsection*{Data preparation}\label{sec:data-prep}

We developed open-source \cite{thibault2023open} pipelines that convert replica spreadsheets (Fig~\ref{fig:typewritten-eg}) into CSV files, using common variables to combine data from different sources (details in Section S1.4 in \nameref{S1_Appendix}). The focal variable is the number of new cases of a specific disease reported in a specific location over a specific time period. CANDID consists of three CSV files, ranging from a minimally processed file offering maximum flexibility in data preparation to a heavily processed file prioritizing convenience; each CSV file corresponds to a stage in the data processing pipeline (Fig S1 in \nameref{S1_Appendix}).

The unharmonized file preserves the raw, digitized data, giving researchers the freedom to apply their own data preparation methods. Descriptors in the unharmonized file use original names (e.g., ``infantile paralysis'' for poliomyelitis before 1924) to minimize historical information loss \cite{torres2022harmonization}. The harmonized file removes low quality data, aggregates some municipal data to provincial levels, and adds harmonized location and disease descriptors that simplify the combination of data from different sources (e.g., poliomyelitis whenever infantile paralysis is reported). Harmonized CSV files are convenient for querying diseases, provinces, and time periods, but they cannot be used directly for analysis due to overlapping incidence values. In data science terms, the harmonized data are not normalized \cite{tidydata,cheng2024general}. For example, the harmonized data include total polio incidence alongside separate values for polio with and without paralysis (see Section S1.4.3 in \nameref{S1_Appendix} for more examples). Such overlapping data are useful for quality control (which we explain in the \nameref{sec:quality-control} section), but data analysis requires removing overlaps to prevent double-counting cases. The normalized file does not contain overlapping incidence values, enabling aggregation without double-counting. When deciding which overlapping records to remove, we generally retain the finest resolution. Continuing our example, we would remove total polio incidence and retain separate records for polio with and without paralysis (see Section S1.4.3 in \nameref{S1_Appendix} for details on removal criteria). For convenience when computing incidence rates, we joined provincial population sizes to the normalized data. All figures in this paper are based on the normalized file. Researchers who wish to make different harmonization or normalization choices can use the upstream files (see Section S1.4.3 in \nameref{S1_Appendix} for details).

\subsection*{Data provenance}\label{sec:data-provenance}

Each record can be traced back to the relevant original scan, replica spreadsheet, and/or script used to produce it, using information in the CSV files (Section S1.4.4 in \nameref{S1_Appendix}). We follow research data management practices by distributing DataCite \cite{datacite_metadata_schema} (version 4.3) metadata with each dataset in the archive. These metadata will make it easier to deposit future versions of CANDID into a research data repository, which we plan to do.

\subsection*{Quality control}\label{sec:quality-control}

We compared sums of incidence values with marginal totals reported both in CANDID sources and in the PHAC portal \cite{totten2019updates}. Discrepancies suggest possible data-entry or scripting errors. We investigated discrepancies and fixed those that appeared to be due to digitization error. These investigations were simplified using our open data provenance tools (see the \nameref{sec:data-provenance} section) and digitized data source replicas (Fig~\ref{fig:typewritten-eg}). We provide full detail on quality control in Section S1.5 in \nameref{S1_Appendix}.

\section*{Results}\label{sec:results}

As of October 2025 CANDID is based on \input{n_sources}\ sources (Table~\ref{tab:sources-table}). Section S1.3 in \nameref{S1_Appendix} provides information on how to access the resulting \input{unharmonized_sample_size}\ unharmonized, \input{harmonized_sample_size}\ harmonized, and \input{normalized_sample_size}\ normalized incidence values for the provinces and territories of Canada (Fig~\ref{fig:map}).

\begin{table}[!ht]
  \centering
  \caption{\textbf{Data sources}. The Frequency column gives the shortest period over which incidence counts were reported for all diseases and locations in the source (if not all disease-location combinations have the shortest period, multiple frequencies are given). Sources that include handwritten data are indicated in the Received As column. Details on these sources are provided in Section S1.1 in \nameref{S1_Appendix}.} 
  \begin{tabular}{ll>{\raggedright\arraybackslash}p{0.11\linewidth}>{\raggedright\arraybackslash}p{0.2\linewidth}>{\raggedright\arraybackslash}p{0.2\linewidth}}  
Years & Provinces & Frequency & Organization & Received As \\ 
  \hline
1903-1947 & Ontario & monthly & Ontario Ministry of Health & Hard copy \\ 
  1910, 1921-1927 & Saskatchewan & monthly & Saskatchewan Department of Public Health & Hard copy \\ 
  1915-1925 & Qu\'{e}bec & monthly & Qu\'{e}bec Ministry of Health and Social Services & Hard copy \\ 
  1924-1955 & All & weekly, monthly & Statistics Canada & Hard copy (with handwriting) \\ 
  1939-1989 & Ontario & weekly & Ontario Ministry of Health & Hard copy (with handwriting) \\ 
  1956-1978 & All & weekly & Statistics Canada & Hard copy \\ 
  1979-1989 & All & 4-weekly & Statistics Canada & Hard copy \\ 
  1990-2001 & All & monthly, quarterly & Health Canada & Hard copy \\ 
  1990-2021 & Ontario & weekly & Public Health Ontario & Spreadsheet \\ 
  2001-2006 & All & quarterly & Canada Communicable Disease Report & PDF \\ 
  2004-2017 & Manitoba & monthly & Public Health Manitoba & PDF \\ 
  2004-2019 & Alberta & weekly & Alberta Health & Spreadsheet \\ 
   \hline
\end{tabular}
\label{tab:sources-table}
\end{table}

\begin{figure}[!h]
  \caption{\textbf{Map of Canada with provinces and territories labelled}. The map is used to show the order of provinces and territories in Fig~\ref{fig:extent-1}, Fig~\ref{fig:extent-2}, Fig~\ref{fig:extent-3}, and the order of provinces in the poliomyelitis example (see the \nameref{sec:results} section). The map also illustrates the grouping of provinces and territories into regions that are used in the whooping cough example (see the \nameref{sec:results} section). This map was prepared using the \texttt{rnaturalearth} \texttt{R} package \cite{rnaturalearth}, which utilized the following public domain shapefile: \url{https://github.com/nvkelso/natural-earth-vector/blob/e08a35c801ac729401e2de9f5eef206031e6a284/10m_cultural/ne_10m_admin_1_states_provinces.shp}.}
  \label{fig:map}
\end{figure}

\newpage 

Fig~\ref{fig:extent-1}, Fig~\ref{fig:extent-2}, and Fig~\ref{fig:extent-3} list the \input{n_basal_diseases}\ diseases that appear in the normalized dataset, and highlight the time periods in which weekly, monthly, or quarterly incidence data were found in each province or territory. Within each disease, vertical placement indicates data availability for a particular province or territory, ordered approximately clockwise as indicated on the map in Fig~\ref{fig:map}. The case numbers for many of these diseases are aggregated from \input{n_diseases}\ ``sub-diseases'' (Section S1.4.2 in \nameref{S1_Appendix}), harmonizing \input{n_historical_diseases}\ unique historical name variants. The stratification of each disease into sub-diseases varied across sources (details in Section S1.4.3 in \nameref{S1_Appendix}).

\begin{figure}[!h]
  \caption{\textbf{Data availability for highly-reported diseases (top \input{n-diseases-page-1} by total reported cases)}. Moderately- and rarely-reported diseases appear in Fig~\ref{fig:extent-2} and Fig~\ref{fig:extent-3}. The diseases are ranked by the total number of cases (right panels), summed over all provinces for which data were obtained. The diseases are ordered with the largest number of cases at the top. Each incidence value, including zeros, is shown as a tiny coloured rectangle. The y-axis labels identify the disease, while the rectangle's length along the x-axis represents the temporal extent. Colours indicate reporting frequency: weekly (blue), monthly (orange), and quarterly (red). Two-weekly data are shown in green but are not included in the legend because they are too few to see without zooming. Three-quarterly data are omitted, as they are few and span most of a year (April-December 1997), providing little additional information on within-year variation. White spaces represent missing data (see Section S1.4.1 in \nameref{S1_Appendix} for details on the varied reasons for missing data). The vertical position within each disease denotes the province/territory, arranged roughly clockwise (east-to-west for provinces and then west-to-east for territories): NL-NS-PE-NB-QC-ON-MB-SK-AB-BC-YT-NT-NU (Fig~\ref{fig:map}). Thin horizontal patterns arise because provinces and territories differ in data availability and reporting frequency: when data are available for only some provinces, or when provinces and territories report at different time scales, the resulting variation appears as horizontal features due to the fixed top-to-bottom ordering of provinces and territories within each disease.}
  \label{fig:extent-1}
\end{figure}

\begin{figure}[!h]
  \caption{\textbf{Data availability for moderately-reported diseases (middle \input{n-diseases-page-2} by total reported cases)}. Highly- and rarely-reported diseases appear in Fig~\ref{fig:extent-1} and Fig~\ref{fig:extent-3}. Please see the caption for Fig~\ref{fig:extent-1} for a full description of all of these plots.}
  \label{fig:extent-2}
\end{figure}

\begin{figure}[!h]
  \caption{\textbf{Data availability for rarely reported diseases (bottom \input{n-diseases-page-3} by total reported cases)}. Highly- and moderately-reported diseases appear in Fig~\ref{fig:extent-1} and Fig~\ref{fig:extent-2}. Please see the caption for Fig~\ref{fig:extent-1} for a full description of all of these plots.}
  \label{fig:extent-3}
\end{figure}


\subsection*{Examples}\label{sec:illustrative-results}

\subsubsection*{Poliomyelitis cases peaked at the same time each year across all provinces}\label{sec:polio-results}

Poliomyelitis incidence was strongly seasonal (upper panel, Fig~\ref{fig:polio-plot}), with national peaks consistently occurring between week \input{polio-min-week-peak} and \input{polio-max-week-peak} each year from \input{polio-min-year-peak} to \input{polio-max-year-peak}, after which the cycles disappeared. These annual cycles were synchronous across provinces, with peak incidence occurring around the same time in each (provincial panels, Fig~\ref{fig:polio-plot}). All national peaks fell between August and October, and most provincial peaks followed this pattern, with only a few outlying province-year combinations (Fig S6 in \nameref{S1_Appendix}). Identifying such spatial synchrony requires incidence data at sub-annual and sub-national scales.

\begin{figure}[!h]
  \caption{\textbf{Weekly poliomyelitis incidence from \input{polio-min-year} to \input{polio-max-year} (square root scale)}. The vertical lines do not indicate the start of each year but mark the week of peak national incidence (top panel) in years with more than 20 cases. Provincial peaks closely align with these national peaks, indicating strong spatial synchrony. This pattern could not have been detected with annual or national data. The introduction of two important vaccination programmes are shown as blue vertical dashed lines. We plot incidence rates per 100,000 to make incidence comparable across provinces and territories, and with other studies. Methods for producing this plot are described in Section S1.6 in \nameref{S1_Appendix}.}\label{fig:polio-plot}
\end{figure}

\subsubsection*{Regional differences in whooping cough incidence}\label{sec:whooping-cough-results}

Aggregating provincial whooping cough data to the national level (Fig~\ref{fig:whooping-plot}) reveals a pattern consistent with previous analyses that lacked provincial data \cite{thommes2020revisiting}. One feature of this pattern is an apparent resurgence of whooping cough in the 1990s (highlighted in Fig~\ref{fig:whooping-plot}). We find that this much-discussed resurgence (e.g., \cite{thommes2020revisiting}) was not uniformly expressed across the country (Fig~\ref{fig:whooping-plot} bottom six panels), and is clearly apparent only in the territories, the prairies, and Qu\'ebec. Throughout the 1990s, yearly cases per $100,000$ peaked at \input{n_on_wc_max_90s} in Ontario, but in the territories the peak was \input{n_territories_wc_max_90s}. The original study \cite{thommes2020revisiting} did not have the sub-national data required to explore these regional differences.

\begin{figure}[!h]
  \caption{\textbf{Regional \cite{SGC2021} differences in average annual whooping cough incidence in Canada over twelve decades (square root scale)}. The national data (top panel) are very similar to the first figure in a review based on different data sources \cite{thommes2020revisiting}. The red region (1990--1999) highlights the first resurgence in national whooping cough incidence since widespread vaccination began in 1943. This plot shows that not all regions peaked in the 1990s, a pattern that could not have been detected with the national data used in the original study. Methods for producing this plot are described in Section S1.7 in \nameref{S1_Appendix}.}
  \label{fig:whooping-plot}
\end{figure}


\section*{Discussion}\label{sec:interpretation}

CANDID complements existing Canadian notification data. The Public Health Agency of Canada (PHAC) provides an online portal \cite{totten2019updates} (\url{https://diseases.canada.ca/notifiable}) with annual, national incidence counts often used in retrospective analyses (e.g., \cite{payne2014hepatitis,gasmi2017emerging,lin2021relaying,lin2021influenza,thommes2020revisiting,golden2024ccdr}). However, these data lack the detail needed to study outbreak patterns, seasonality, or geographic variation. Our sub-annual, sub-national data enable research on patterns of variation within years and across provinces in Canada, and facilitate comparisons with U.S. data \cite{van2018project}. While Public Health Ontario provides online monthly data since 2012 \cite{phoquery}, our archive includes weekly Ontario data (1990--2021) and extends back before 1924, including Ontario (1903), Saskatchewan (1910), and Québec (1915). By consolidating federal and provincial sources (Section S1.1 in \nameref{S1_Appendix}) into a standardized format (Section S1.4 in \nameref{S1_Appendix}), we simplify integration of new data, enabling researchers to focus on analysis rather than curation.

Our project parallels Project Tycho \cite{van2013contagious}, which curated weekly U.S. incidence data and whose impact is summarized by \cite{van2018project}. Unlike Tycho, we open-sourced our data preparation pipelines to enable community-driven quality improvements. These pipelines include scans of original documents, spreadsheet replicas, and scripts to convert them into tidy CSV files. To our knowledge, no other studies publish such replicas (see the \nameref{sec:data-entry} section), which help identify and correct data-entry errors. Our open-science approach enables researchers to trace incidence counts back to original sources (Section S1.4.4 in \nameref{S1_Appendix}) and improve data quality over time.

A well-established feature of poliomyelitis is its seasonal pattern, with epidemics in North America typically occurring in summer. Ref. \cite{martinez2015unraveling} attributed earlier epidemics in southern U.S. states to higher transmission rates. Our data extend this latitudinal gradient northward: epidemics across Canada tended to occur later (August--October) than those in the U.S. (May--July). However, we find no substantial differences in timing among Canadian provinces from 1934 to 1960, despite large variation in climate.

\subsection*{Future directions}\label{sec:future}

In this initial release of CANDID, we focused on sub-annual and sub-national disease incidence, with plans to expand further. First, we will include data stratified by age, sex, and municipality for available time ranges, provinces, and diseases. Age data, for example, can be critical for estimating the impact of vaccination programmes in childhood diseases (e.g., \cite{wilkinson2022impact}). Second, we will extend the dataset's time range, disease coverage, and geographic detail as finer-scale or corrected data become available. Third, we will curate population-level information useful for epidemiological analyses, such as birth rates, mortality, vaccination, and school-term dates. Beyond expansion, we will address open questions in Canadian infectious disease history. For example, we will test whether regional differences in vaccination programmes could explain spatial variation in the size of the 1990s resurgence of whooping cough (Fig~\ref{fig:whooping-plot}). Finally, we will explore AI-powered optical character recognition, using our archive as a training dataset to enhance its efficiency and accuracy.

\subsection*{Logistical challenges}\label{sec:logistics}

The process of assembling CANDID highlighted logistical obstacles that are likely to arise in other countries seeking to build comparable historical surveillance datasets. The first challenge was locating and obtaining source material. Historical records were dispersed among federal and provincial publications issued under evolving titles and formats, with many available only as hard copies in archives or libraries. Coordinated searches, correspondence with data stewards, and formal data requests to multiple agencies and libraries were required. These challenges are described in more detail in Section S1.1 in \nameref{S1_Appendix}.

A second challenge was manual data entry. Optical character recognition could not accurately extract numerical tables from scans, necessitating full manual transcription. Coordinating several data enterers required standardized templates, shared conventions for unclear entries, and frequent communication to ensure consistency across spreadsheets. Looking ahead, advances in artificial intelligence may mitigate some of these difficulties, as noted in our future plans (see the \nameref{sec:future} section), although many questions remain.

A third challenge was harmonizing information drawn from sources that differed widely in format and structure. Converting these diverse formats into consistent, machine-readable CSV files and resolving overlaps across locations, time scales, disease hierarchies, and data sources required careful sequencing of processing steps and version control across multiple repositories. These challenges are detailed in Section S1.4 in \nameref{S1_Appendix}.

Finally, quality control involved iterative cross-checks across diseases, jurisdictions, and time scales. Addressing discrepancies meant revisiting earlier stages of data entry and processing, emphasizing the importance of reproducible pipelines. These challenges illustrate the logistical scale of constructing a comprehensive historical surveillance archive. By documenting our workflows and tools openly, we have worked for CANDID to serve as both a resource and a practical starting point for researchers undertaking similar digitization efforts elsewhere.

\subsection*{Limitations}\label{sec:limitations}

Under-reporting is a known limitation of surveillance data \cite{van2013contagious,gibbons2014measuring}. Correcting for it requires disease- and context-specific methods that rely on supplementary data (e.g., serological surveys, case-fatality ratios, demography) and modelling \cite{mangen2011current}. Given the number of diseases and years covered, such corrections are beyond our scope. Our contribution is to make data available in a form that supports future work, including efforts to address under-reporting. As one step in that direction, we distinguish true zero case counts from data that were unreported, based on information available in the original sources (see Section S1.4.1 in \nameref{S1_Appendix}).

In addition to under-reporting, two other factors complicate the interpretation of these incidence data: evolving sub-disease hierarchies and inconsistent time scales. First, changes in how diseases are classified and reported over time affect comparability, as the level of aggregation can vary year to year (e.g., the normalized dataset includes 37 meningitis sub-diseases, with 1--15 reported in any given year; Fig S3 in \nameref{S1_Appendix}). Second, variation in reporting frequency (e.g., weekly, monthly, quarterly; Fig~\ref{fig:extent-1}, Fig~\ref{fig:extent-2}, Fig~\ref{fig:extent-3}) hinders the construction of evenly spaced time series, which are often needed for modeling and visualization.

Historical gaps in CANDID persist due to surveillance program changes (e.g., chickenpox was not notifiable from 1959--1985) and incomplete source coverage, particularly outside Québec, Ontario, and Saskatchewan before 1924. These gaps range from missing weeks (e.g., lost book pages) to illegible handwritten records (see the \nameref{sec:quality-control} section). Although transcription and coding errors cannot be ruled out, we compared reported subtotals with their marginal totals and corrected all identified discrepancies for our primary example diseases (whooping cough, poliomyelitis). By releasing open data pipelines, we aim to enable similar checks across all diseases and to support collective efforts to improve the completeness and reliability of CANDID over time (details in Section S1.5 in \nameref{S1_Appendix}).

\subsection*{Conclusion}

More than a century of infectious disease surveillance in Canada has produced a valuable record of epidemic patterns that has been largely unexploited, but can now be easily accessed. Comprehensive sub-annual and sub-national Canadian infectious disease incidence data have previously been unavailable. Similar data from other countries have been critical to establishing the foundations of epidemiological modelling and continue to push the field forward (e.g., \cite{bartlett1960critical,london1973recurrent,anderson1982directly,fine1982measles,grenfell1985pertussis,AndeMay91,Tien+11,KrylEarn20,Earn+20}).

CANDID makes it possible to study variation in disease incidence within years and across provinces, with applications ranging from infectious disease research to broader interdisciplinary work. For example, it can be used to assess how incidence in Canada relates to socio-economic factors such as urbanization and wealth inequality. The dataset also supports public health planning by situating recent outbreaks and epidemics within their historical context.

In principle, it should be straightforward to keep the archive up to date, but doing so will require the cooperation of provincial and territorial public health agencies, which have released \emph{less} data publicly since strictly digital data collection began in the 1990s. We contacted all these agencies but were able to obtain recent weekly data from only two provinces. While we recognize that agencies may face practical constraints, such as the staff time required to prepare and maintain public releases, we encourage Canadian governments to support routine public access to weekly, aggregated counts of infectious disease notifications.


\section*{Supporting information}

\paragraph*{S1 Appendix}\label{S1_Appendix}
{\bf Methodological Details.} This supporting information details the historical and methodological framework for compiling and preparing Canadian notifiable infectious disease incidence data. It explains the sources and evolution of the Canadian Notifiable Disease Surveillance System (CNDSS), including federal and provincial publications, and describes the process of locating, digitizing, and cleaning these historical records. Data entry procedures are described along with the individuals who did the work. Quality control procedures are described, including internal consistency checks across time scales, locations, and disease hierarchies, as well as comparisons among data sources. Finally, the document outlines specific analytical methods for diseases like polio and whooping cough.



% % Include only the SI item label in the paragraph heading. Use the \nameref{label} command to cite SI items in the text.
% \paragraph*{S1 Fig.}
% \label{S1_Fig}
% \textbf{Bold the title sentence.} Add descriptive text after the title of the item (optional).

% \paragraph*{S2 Fig.}
% \label{S2_Fig}
% \textbf{Lorem ipsum.} Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.

% \paragraph*{S1 File.}
% \label{S1_File}
% \textbf{Lorem ipsum.}  Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.

% \paragraph*{S1 Video.}
% \label{S1_Video}
% \textbf{Lorem ipsum.}  Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.

% \paragraph*{S1 Appendix.}
% \label{S1_Appendix}
% \textbf{Lorem ipsum.} Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.

% \paragraph*{S1 Table.}
% \label{S1_Table}
% \textbf{Lorem ipsum.} Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.

\section*{Acknowledgments}

We are deeply thankful to Alberta Health and Public Health Ontario for providing us with recent weekly incidence data. Research assistants Jen Freeman, Frank Jin, Ronald Jin, and Steven Lee wrote code that we used during this project. Research assistants Jeanne Lin, Saul Widrich, Qinxian Zhu, Claire Lees, and Julia Maja entered some of the data. Research assistants Maya Earn, Arielle Earn, and Elizabeth O'Meara found, organized, and scanned source documents.  We appreciate the enthusiastic encouragement we received from Caroline Colijn, Michael Li and many other colleagues in the Canadian Network for Modelling Infectious Diseases (CANMOD).

\nolinenumbers

% Please compile your BiBTeX database using the "plos2025.bst" BibTeX style.
% This file is part of the current package.
% A sample BibTeX file is also included as "plos_bibtex_sample.bib".
%
% or
%
% Type in your references following Vancouver style and reference formatting instructions
% available at https://journals.plos.org/plosone/s/submission-guidelines#loc-references
% \begin{thebibliography}{}
% \bibitem{}
% Text
% \end{thebibliography}

\bibliography{bibliography}

\end{document}

